travis.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -e
 
+# Check that the code has been formatted correctly.
+test -z "$(gofmt -l $(find -name '*.go' | grep -v /vendor/))"
```

Comment:
you can do this but it's doubtless slower than simply
gofmt -s -l *.go pkg cmds
?
I just tried that and it works fine.